modules/server/src/main/scala/navigate/server/Systems.scala

```diff
@@ -38,13 +39,45 @@ object Systems {
 
     def buildTcsSouthController: Resource[F, TcsSouthController[F]] =
       if (conf.systemControl.tcs === ControlStrategy.FullControl)
-        TcsEpicsSystem.build(epicsSrv, tops).map(new TcsSouthControllerEpics(_, conf.ioTimeout))
+        for {
+          tcs <- TcsEpicsSystem.build(epicsSrv, tops)
+          p1  <- WfsEpicsSystem.build(epicsSrv,
+                                      "PWFS1",
+                                      readTop(tops, NonEmptyString.unsafeFrom("pwfs1"))
```

Comment:
you can do `"pwfs1".refined` importing `lucuma.refined.*`

Reply:
Las cosas que uno aprende